src/components/buttons/Swipe.tsx

```diff
@@ -119,38 +115,23 @@ const SwipeButton = (props) => {
 
   return (
     <AnimatedLinearGradient
-          style={styles.swipeCont}
-          start={[1, 0]}
-          end={[0, 1]} colors={COLORS.buttonGradient}    >
+      style={styles.swipeCont}
+      colors={["rgba(183, 0, 76, 0.3)", "rgba(19, 63, 219, 1)"]}
+      start={[1, 0]}
+      end={[0, 1]}
+    >
       <PanGestureHandler onGestureEvent={animatedGestureHandler}>
         <Animated.View style={[styles.swipeable, AnimatedStyles.swipeable]}>
-          <AntDesign name="arrowright" size={24} color={COLORS.darkBlue} />
+          <AntDesign name="arrowright" size={24} color="#002B4E" />
         </Animated.View>
       </PanGestureHandler>
       <Animated.Text style={[styles.swipeText, AnimatedStyles.swipeText]}>
-        {props.title}
+        Swipe to confirm
       </Animated.Text>
     </AnimatedLinearGradient>
   );
 };
 
-/**
```

Comment:
Let this remain. They are for documentation purposes. Someone new using the button does not have to got to the code to check the props being used. They serve as defaults.